fv3core/pace/fv3core/stencils/fv_dynamics.py

```diff
@@ -480,10 +484,11 @@ def _compute(self, state: DycoreState, timer: pace.util.Timer):
                         self._timestep,
                     )
                 if last_step:
+                    da_min: float = self._get_da_min()
```

Comment:
Does the type hint do anything?

```suggestion
                    da_min = self._get_da_min()
```

Reply:
Yes. see above